pandas/tests/extension/test_string.py

```diff
@@ -156,13 +156,9 @@ def test_reduce_series_numeric(self, data, all_numeric_reductions, skipna):
 
 
 class TestMethods(base.BaseMethodsTests):
-    @pytest.mark.skip(reason="returns nullable")
```

Comment:
yah really ought to pull the trigger on #44692 and just get internal consistency here